#!/usr/bin/env python

PACKAGE = "kinect_tomato_searcher"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("offset_x", double_t, 0, "calibration of linear x", 0.010, 0, 1)
gen.add("offset_y", double_t, 0, "calibration of linear y", 0.000, 0, 1)
gen.add("offset_z", double_t, 0, "calibration of linear z", 0.000, 0, 1)

exit(gen.generate(PACKAGE, "kinect_tomato_searcher", "Calibration"))
